client/reader/like-helper.jsx

```diff
@@ -4,13 +4,28 @@ import {
 	isDiscoverSitePick,
 } from 'calypso/reader/discover/helper';
 
+/**
+ * True if likes are explicitly disabled.
+ * False if likes are enabled or the property does not exist.
+ *
+ * @param {object} post
+ * @returns {boolean} True if likes are explicitly disabled
+ */
+function isLikesDisabled( post ) {
```

Comment:
I personally find that it is a bit harder to mentally process a negated predicate name (`isLikesDisabled`). Furthermore the field name being used here is actually the positive version of the predicate (`likes_enabled`).

What if we have something like this?

```
function isLikesEnabled( post ) {
  return post?.likes_enabled ?? true;
}
```

Here, it is also clear that the supposed "default value" of this field is `true`.

Reply:
ahh nice, I think that is clearer while still maintaining the behavior, I couldn't think of how to achieve that, nice improvement